genrl/deep/common/logger.py

```diff
@@ -73,7 +75,7 @@ def __init__(self, logdir: str):
         self.lens = []
         self.maxlen = 0
 
-    def write(self, kvs: Dict[str, Any]) -> None:
+    def write(self, kvs: Dict[str, Any], _) -> None:
```

Comment:
Don't use an underscore here. You should use `x_axis` like in the other loggers.